### PR TITLE
CTPAT string types and separation of IOR

### PR DESCRIPTION
**Pull request by nissimsan**
This: 
- adds string types to enumerated fields
- separates importer of record to a dedicated type

### CODE REVIEW

**Review by BenjaminMoe — APPROVED** [review]

File: docs/openapi/components/schemas/common/CBPImporterOfRecord.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+$linkedData:
+  term: CBPImporterOfRecord
+  '@id': https://w3id.org/traceability#CBPImporterOfRecord
+title: CBP Importer Of Record
+description: US CBP importer of record identifier and itentifier type (IRS, SSN or CBP).
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CBPImporterOfRecord.yml:5]:
```suggestion
description: US CBP importer of record identifier and identifier type (IRS, SSN, or CBP).
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: docs/openapi/components/schemas/common/CTPAT.yml [hunk 2]
```diff
@@ -64,23 +64,34 @@ properties:
         $linkedData:
           term: iataCarrierCode
           '@id': https://onerecord.iata.org/cargo/Company#airlineCode
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/CTPAT.yml:66]: I know this isn't part of this PR but this probably wants to reference ` https://onerecord.iata.org/Carrier#airlineCode` as company#airlinecode isn't a thing

File: docs/openapi/components/schemas/common/CTPATMember.yml [hunk 3]
```diff
@@ -48,23 +48,13 @@ properties:
     $linkedData:
       term: iataCarrierCode
       '@id': https://onerecord.iata.org/cargo/Company#airlineCode
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/CTPATMember.yml:50]: Similar to above comment

**Review by mkhraisha — CHANGES_REQUESTED** [review]

File: docs/openapi/components/schemas/common/CTPAT.yml (hunk 2 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/CTPAT.yml:66]: No, wait, `https://onerecord.iata.org/Carrier#airlineCode` _is_ a thing. The IATA ontology explicitly calls that out: 
image

There's just an in consistency with how the ontology is published. I called that out more than half a year ago on the IATA github: 
https://github.com/IATA-Cargo/ONE-Record/issues/149

I'm open to arguments for going with the double-# URI instead. But I feel pretty confident about sticking with my earlier choice: 
1. The IATA spec explicitly says the IRI is `https://onerecord.iata.org/Carrier#airlineCode` 
2. Double-# URIs are not illegal, but discouraged (second # must be interpreted as a literal), further indicating that's the bug and not IATA's actual intention.
Pls open a separate issue if you think this warrants further discussion; I don't think it does.

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**